week44/문자열 내 마음대로 정렬하기_hyeinlee725.py

```diff
@@ -0,0 +1,4 @@
+def solution(strings, n):
+    strings.sort()
+    answer = sorted(strings, key = lambda x : x[n])

```

Comment:
```suggestions
answer = sorted(strings, key = lambda x : (x[n], x))
```
key값으로 조건을 여러개 줄 수 있어서 이렇게 바꾸면 `strings.sort()`를 따로 안 해줄 수 있습니다.